### PR TITLE
Add format to verify target

### PR DESCRIPTION
**Pull request by awgreene**
This commit updates the verify target to include the
format target as a prerequisite. This commit also
includes changes introduced after running the format
target.

### CODE REVIEW

File: crds/zz_defs.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Code generated by go-bindata. (@generated) DO NOT EDIT.
 
- //Package crds generated by go-bindata.// sources:
```
Comment (timflannagan) [inline crds/zz_defs.go:3]: Do we want to be formatting generated files? Can we adjust the fmt target to filter out certain files/directories/etc.?

Comment (estroz) [reply crds/zz_defs.go:3]: IMO if the generator isn't formatting Go files it has a bug, so I'm fine with formatting everything (with the exception of vendor dirs).

**Pull request merged**